guides/common/modules/proc_configuring-http-proxy-to-connect-to-cdn.adoc

```diff
@@ -1,29 +1,21 @@
 [id="configuring-http-proxy-to-connect-to-cdn_{context}"]
 = Configuring the HTTP Proxy to Connect to Red Hat CDN
 
-To configure subscription-manager with an HTTP proxy, follow the procedure below.
+.Prerequisites
+
+Your network gateway and the HTTP proxy must allow access to the following hosts:
+include::snip_host-names-for-http-proxy.adoc[]
+
+To configure the subscription-manager with the HTTP proxy, follow the procedure below.
 
 .Procedure
 
-. On the network gateway and the HTTP Proxy, enable TCP for the following host names:
```

Comment:
I have merged line `8` (old) with the line `6` (new).  Need the technical validation.